xlators/debug/io-stats/src/io-stats.c

```diff
@@ -292,7 +292,7 @@ is_fop_latency_started(call_frame_t *frame)
         begin = &frame->begin;                                                 \
         end = &frame->end;                                                     \
                                                                                \
-        elapsed = gf_tsdiff(begin, end) / 1000.0;                              \
+        elapsed = gf_tsdiff(begin, end);                                       \
```

Comment:
Not sure why! but with this patch, I see that 'elapsed' is 0, which means, the next line gave SIGFPE... Please add a check here.

Reply:
Sure I'll add a check